src/components/UI/MyFriendsCard.jsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Card, CardContent, CardMedia, styled } from '@mui/material'
+import React from 'react'
+
+import CardImage from '../../assets/images/Card-Image.png'
+
+const myFriends = [
+   {
+      name: 'Annette Black',
+      textWish: 'Желаний',
+      textHolidays: 'Праздников',
+      id: '1',
+   },
+   {
+      name: 'Annette Black',
+      textWish: 'Желаний',
+      textHolidays: 'Праздников',
+      id: '2',
+   },
+]
+
+export const MyFriendsCard = ({ wish = '12', holidays = '10' }) => {
```

Comment:
we don't get wish and holidays from props. Please add these to objects in array. Because when we send request to backend we get only array with objects and there i think we get wish and holiday from each person